boards/nxp/frdm_mcxn947/frdm_mcxn947_mcxn947_cpu0_defconfig

```diff
@@ -10,6 +10,10 @@ CONFIG_SERIAL=y
 CONFIG_UART_INTERRUPT_DRIVEN=y
 CONFIG_GPIO=y
 
+# The existing SAI diver cannot initialize the PLL on the board,
+# so the PLL settings will not be performed in the driver.
+CONFIG_I2S_HAS_PLL_SETTING=n
+
```

Comment:
```suggestion
```
Sorry for the confusion, but this Kconfig should be deselected in the SOC file, not here at the board level.  Can you please put the lines below back in soc/nxp/mcx/mcxn/Kconfig.defconfig?
```
# The existing SAI diver cannot initialize the PLL on the board,
# so the PLL settings will not be performed in the driver.
config I2S_HAS_PLL_SETTING
	default n
```

Reply:
done